client/src/globals.js

```diff
@@ -83,3 +83,21 @@ export function keyPressSubmit(e) {
     if (submit) submit.click()
   }
 }
+
+export async function shareSlack(that, path, apiCall) {
+  that.setState({ disableSlack: true })
+  let succ = null
+  try {
+    const { success } = await apiCall('GET', `/slack/notify/${path}`)
+    succ = success
+  } catch (error) {
+    // Do nothing
```

Comment:
Really?

Reply:
Yes this is intentional

Reply:
Is the `try` expected to fail on normal usage?
If not should probably at least add `console.error()` for it.

Reply:
it's expected and catches unauthorized action, also shows error toast on UI